dbc2val/test/test_example_mapping/test_example_mapping.py

```diff
@@ -76,55 +75,56 @@ def test_parking_lock():
     # CurrentGear tests
     vss_mapping = dbc_mapping[park_lock_index]
     assert vss_mapping.vss_name == "Vehicle.Powertrain.Transmission.IsParkLockEngaged"
-    assert isinstance(vss_mapping.transform,dict)
+    assert isinstance(vss_mapping.transform, dict)
     assert "mapping" in vss_mapping.transform
     mapping_list = vss_mapping.transform["mapping"]
     assert len(mapping_list) == 4
     # And now just check some items
     assert mapping_list[0]["from"] == "DI_GEAR_D"
-    assert mapping_list[0]["to"] == False
+    assert mapping_list[0]["to"] is False
     assert mapping_list[1]["from"] == "DI_GEAR_P"
-    assert mapping_list[1]["to"] == True
+    assert mapping_list[1]["to"] is True
 
     # Now test some transforms
     result = vss_mapping.transform_value("DI_GEAR_D")
-    assert result == False
+    assert result is False
     result = vss_mapping.transform_value("DI_GEAR_P")
-    assert result == True
-    
+    assert result is True
+
     # For non successful mapping old value shall be kept and None returned
     result = vss_mapping.transform_value("Something undefined")
-    assert result == None
+    assert result is None
+
 
 def test_vehicle_speed():
     dbc_mapping = mapper["DI_uiSpeed"]
     assert len(dbc_mapping) == 2
```

Comment:
do we want is here instead of == like you changed at the lines above (True or False)?

Reply:
`is` compares if two objects are the same. `==` if two objects are equal. It seems `is` is preferred when comparing with `None`, and at least some linters wants it for comparison with `True` and `False` as well. But for integers it only seems to work for smaller values where type object is cached. So my understanding is that `==` is recommended for integer comparison

https://realpython.com/courses/python-is-identity-vs-equality/
https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is